Tools/Scripts/libraries/webkitcorepy/webkitcorepy/autoinstall.py

```diff
@@ -385,7 +385,8 @@ def install(self):
                 else:
                     # We might not need setup.py at all, check if we have dist-info and the library in the temporary location
                     to_be_moved = os.listdir(temp_location)
-                    if self.name not in to_be_moved and any(element.endswith('.dist-info') for element in to_be_moved):
+                    has_module = self.name in to_be_moved or '{}.py'.format(self.name) in to_be_moved
```

Comment:
Pedantically, this is `has_module_or_package`, but that's probably overly minutiae. 🙃